.github/workflows/linux.yml

```diff
@@ -472,23 +501,35 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: aks-linux-4-cores
+    container:
+      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+      volumes:
+        - /mount/caches:/mount/caches
     env:
-      OPENVINO_REPO: ${{ github.workspace }}/openvino
-      OPENVINO_CONTRIB_REPO: ${{ github.workspace }}/openvino_contrib
-      INSTALL_DIR: ${{ github.workspace }}/install
-      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
-      SAMPLES_INSTALL_DIR: ${{ github.workspace }}/install/samples
-      LAYER_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/layer_tests
-      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
-      BUILD_DIR: ${{ github.workspace }}/build
-      OV_TEMP: ${{ github.workspace }}/openvino_temp
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      OPENVINO_REPO: /__w/openvino/openvino/openvino
+      OPENVINO_CONTRIB_REPO: /__w/openvino/openvino/openvino_contrib
+      INSTALL_DIR: /__w/openvino/openvino/install
+      INSTALL_TEST_DIR: /__w/openvino/openvino/install/tests
+      SAMPLES_INSTALL_DIR: /__w/openvino/openvino/install/samples
+      LAYER_TESTS_INSTALL_DIR: /__w/openvino/openvino/install/tests/layer_tests
+      MODEL_HUB_TESTS_INSTALL_DIR: /__w/openvino/openvino/install/tests/model_hub_tests
+      BUILD_DIR: /__w/openvino/openvino/build
+      DATA_PATH: /__w/openvino/openvino/testdata
+      MODELS_PATH: /__w/openvino/openvino/testdata
+      OV_TEMP: /__w/openvino/openvino/openvino_temp
```

Comment:
we don't need all these vars on tests stage:
- DATA_PATH
- MODELS_PATH
- BUILD_DIR
and some others.